lib/tasks/subject.rake

```diff
@@ -139,7 +139,7 @@ namespace :birs do
 
     ams_subject_codes.each do |ams_subject|
       puts "Adding AMS Subject: #{ams_subject[:title]}"
-      AmsSubject.create(code: ams_subject[:code], title: ams_subject[:title])
+      AmsSubject.find_or_create_by(code: ams_subject[:code], title: ams_subject[:title])
```

Comment:
For an existing AmsSubject with code: "code1", and title: "03 - General Relativity", won't the find fail because "code1" != 03-XX ? And in this case, won't a new subject be added, causing two subjects with the same title to appear in the user interface?